.github/workflows/pr-actions.yml

```diff
@@ -0,0 +1,160 @@
+name: PR actions
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions: read-all
+
+env:
+  COMMENT: ${{ github.event.comment.body }}
+  PR_NUM: ${{ github.event.issue.number }}
+  USER_EMAIL: 107717825+opentelemetrybot@users.noreply.github.com
+  USER_NAME: opentelemetrybot
+  MAX_PATCH_SIZE_KB: 1024
+
+jobs:
+  generate-patch:
+    name: Run fixer and generate patch (untrusted)
+    runs-on: ubuntu-latest
+    if: |
+      github.event.issue.pull_request &&
+      startsWith(github.event.comment.body, '/fix:')
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
this is the only nit I can find, which could give someone the ability to add labels/milestones/projects/etc to their (own) PR, but I think that's probably an ok compromise

the GitHub App requested in https://github.com/open-telemetry/community/issues/2746 does something similar, but doesn't require this permission since it uploads the diff to their service

but it is an external GitHub App, which brings its own security considerations

Reply:
I'd love to see a label spamming attack :D

Reply:
the only concern I think is if a repo has some label-driven automation or bypasses

Reply:
True that. I don't think that's our case though, right @chalin ?

Reply:
I wonder if the permission is even necessary in this step (sorry, I rendered some of it with AI + copy&paste from the existing one). We can drop it and run without first